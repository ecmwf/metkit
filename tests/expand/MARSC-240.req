retrieve,
DOMAIN=G,LEVTYPE=DP,DATE=20070601,TIME=0000,STEP=288,PARAM=145.151,CLASS=OD,TYPE=OF,STREAM=OCEA,EXPVER=0001,NUMBER=0,SYSTEM=2,METHOD=1,PRODUCT=TIMS,SECTION=Z,LEVELIST=0.000,LATITUDE=-9.967,RANGE=264,
target="reference.0DY4f2.data"