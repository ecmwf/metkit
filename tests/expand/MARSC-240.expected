RETRIEVE,
    CLASS      = OD,
    TYPE       = OF,
    STREAM     = OCEA,
    EXPVER     = 0001,
    REPRES     = SH,
    LEVTYPE    = DP,
    LEVELIST   = 0,
    PARAM      = 151145,
    DATE       = 20070601,
    TIME       = 0000,
    RANGE      = 264,
    STEP       = 288,
    NUMBER     = 0,
    DOMAIN     = G,
    SYSTEM     = 2,
    METHOD     = 1,
    PRODUCT    = TIMS,
    SECTION    = Z,
    LATITUDE   = -9.967,
    TARGET     = reference.0DY4f2.data 